modules/nw-multus-configure-dualstack-ip-address.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: PROCEDURE 
+
+[id="nw-multus-configure-dualstack-ip-address_{context}"]
+= Creating a configuration for assignment of dual-stack IP addresses dynamically
+
+Dual-stack IP address assignment can be configured with the `ipRanges` parameter for:
+* IPv4 addresses
+* IPv6 addresses
+* multiple IP address assignment 
+
+.Procedure
+
+. Set type to whereabouts.
+. Use `ipRanges` to allocate IP addresses.
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    type: Raw
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-dual-stack",
+       "cniVersion": "0.3.1,
+       "type": "bridge",
+       "ipam": {
+         "type": "whereabouts",
+         "ipRanges": [
+                  {"range": "192.168.10.1/24"},
+                  {"range": "2001:db8::/64"}
+              ]
+       }
+      }
+
+
+----
+
+. Attach network to a pod. For more information, see "Adding a pod to an additional network".
+
+. Verify that all IP addresses are assigned.
+
+Run the following two commands to ensure the IP adresses are assigned as meta-data.
```

Comment:
there's a single command below.

furthermore, it depends on the workload image - i.e. it will fail is the customer the image is using does not have the `ip` command baked in.

Finally, I don't understand the "assigned as meta-data" part of the sentence.

Reply:
@dougbtv can you answer the "assigned as meta-data" part?

Reply:
Fixed the two command bit- does the meta-data part need to change?